src/openforms/forms/models/form_authentication_backend.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from openforms.authentication.fields import (
+    BackendChoiceField as AuthenticationBackendChoiceField,
+)
+from openforms.authentication.registry import register as auth_register
+
+from . import Form
+
+
+class FormAuthenticationBackend(models.Model):
+    form = models.ForeignKey(
+        Form,
+        on_delete=models.CASCADE,
+        related_name="auth_backends",
+    )
+    backend = AuthenticationBackendChoiceField(_("authentication backend"))
+    options = models.JSONField(
+        _("authentication backend options"),
+        blank=True,
+        null=True,
+    )
```

Comment:
my earlier remark about `default=dict` is wrong, not sure where I saw that, maybe I'm confusing it with another PR!